src/app/tabs.component.test.js

```diff
@@ -0,0 +1,23 @@
+describe('TabsCtrl', () => {
+  let $componentController;
+  let scope;
+
+  beforeEach(angular.mock.module('AngularJSApp'));
+  beforeEach(
+    angular.mock.inject((_$componentController_, $rootScope) => {
+      scope = $rootScope.$new(true);
+      $componentController = _$componentController_;
+    })
+  );
+
+  it('should be defined', () => {
+    const ctrl = $componentController('appTabs', { $scope: scope }, { $digestCount: 0 });
+    scope.$watch('ctrl.$digestCount', () => ctrl.$digestCount++);
```

Comment:
This is also not right. I don't understand the purpose of it, but it does not seem to belong in this test :grin: